.travis/run_tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [[ $TRAVIS ]]; then
+    cd "$TRAVIS_BUILD_DIR" || exit 2
+
+    # Check modified PHP files with PHP's internal syntax checker
+    git diff --name-only HEAD^ | grep '\.php$' | xargs -r -n 1 php -l || exit 1
+
+    # Run test suite
+    vendor/bin/phpunit tests/DoctrineTestSuite1.php
+else
+    echo 'ABORTED: NOT RUNNING ON TRAVIS'
+    exit 2
```

Comment:
Why an exit code of 2?

Reply:
Because `1` would indicate test failure, `2` is a standard exit code for "usage error".

Reply:
Doesn't Travis treat any non-zero exit code in the `script` section as a fail?

Reply:
Yes indeed, but this is to catch the case of a user running the script manually, Travis wouldn't get here.